test/input-test.jl

```diff
@@ -3,7 +3,7 @@
     freq = rand(1:100)
     dt = rand() * (1/freq)
     rate = freq * dt
-    input = ConstantRate(rate)
+    input = ConstantRate(rate; rng = MersenneTwister())

```

Comment:
Can you change this test so that it does it for both w/o `rng` specified and w/ `rng = Random.MersenneTwister()`?